CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [23.3.1] - 2019-07-02
+### Fixed
+- Fennec Release cannot be shipped off mozilla-esr68
```

Comment:
Nit: this somewhat contradicts the title & logic from the PR

Reply:
Good catch. Fixed.